pkg/web/openapi/swagger-ui/api/v1/api.swagger.json

```diff
@@ -155,6 +155,45 @@
         ]
       }
     },
+    "/v1/connectors/validate": {
+      "post": {
+        "operationId": "ConnectorService_ValidateConnector",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/v1ValidateConnectorResponse"
+            }
+          },
+          "500": {
```

Comment:
WDYT about returning something other than 500? This feels like a client error, not a server-side one.

Reply:
That would make sense, the problem is we can't differentiate a validation error from any other error right now, the plugin `Configure` method either succeeds or fails. We need to first generally define the mechanism how we could make Conduit aware of specific plugin errors (e.g. marshal errors into a certain format and include an error code or something).

Reply:
What we could do is just return 400 (and the corresponding gRPC error code) in case the validation failed.